receivings.json

```diff
@@ -1,44 +1,45 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "collection of receivings",
+  "description": "A collection of receivings",
   "type": "object",
   "properties": {
-    "receivings": {
-      "description": "an array of receivings",
-      "id": "receivings",
+    "receivingItems": {
+      "description": "List of receivings",
+      "id": "receivingItems",
       "type": "array",
-      "items": {
+      "receivingItems": {
```

Comment:
I believe this is supposed to be called just "items" for it to identify it is an array and generate the array Object. Can you please crosscheck